### ISSUE TITLE
Send out of stock message when last item is bought

### ISSUE THREAD

**Issue by Phoenix616** [ChestShop-authors/ChestShop-3#206]
Currently the out of stock message is only send to the owner when a player tries to buy another item from the shop when it is out of stock, this would make more sense to send directly once the shop is out of stock (and wasn't automatically removed) to directly indicate this information to the owner so he can restock.

**Comment by danisaacs** [comment]
Good idea.  perhaps even as a subcommand.  Just as /csn history shows last n transactions, /csn stock could show last n items to be out of stock.

I'm as likely to see an out of stock chat message currently as I am to see one sent when the chest is empty.  So having that info handy via a command would be better.

**Comment by Phoenix616** [comment]
This is about the normal out of stock message in ChestShop, for ChestShopNotifier related suggestions please use the corresponding repository and issue tracker.

**Comment by danisaacs** [comment]
Ah. my apologies, didn't realize they were separate plugins.
